GVFS/GVFS.Build/GenerateUnifiedInstallScript.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.IO;
+using System.Text;
+
+namespace GVFS.PreBuild
+{
+    public class GenerateUnifiedInstallScript : Task

```

Comment:
The current install scripts have absolute paths _as built_ on the build agent. They're not portable. This generates a script with relative paths to the installers, so it can be invoked from the portable build drop.

Reply:
However it's not exactly consistent with how we're doing this on macOS. I'll look at reconciling those behaviors in the morning.

Reply:
After reviewing options, I slightly prefer how Windows is accomplishing this. I'll look at addressing the macOS behavior in a subsequent PR.

Reply:
However, I did teach the existing `GenerateInstallScripts` task how to also generate the full product installer, and had them reference the installers in the output directory, so it's cleaner now.